### DISCUSSION TITLE
Serverless deployment failing

### DISCUSSION THREAD

**Discussion by AnishLushte07** [serverless/serverless#11559] [Q&A]
Deployment is failing with following error:
Environment: linux, node 12.22.12, framework 3.25.0, plugin 6.2.2, SDK 4.3.2
Error:  **Request is missing Authentication Token**

Same branch deployment worked 2 days back. Can someone please help me understating reason of failure.

**Comment by dantetwc** [comment]
Same here

**Comment by andrmoel** [comment]
Related: https://github.com/serverless/serverless/issues/11561
